web-console/knikubevirt/Create-vm/create-vm.md

```diff
@@ -0,0 +1,189 @@
+
+As soon as the OS, Flavor, or Workload profile fields are modified, the “Create new template from configuration” checkbox will no longer be disabled.
+
+## template
+
+![template chosen](img/Step-1-basic-template.png)
+
+Template automatically loads the first (alphabetical) template and fills in the Provision Source, Operating System, Flavor, and Workload Profile. Only Flavor can be changed by the user between the template-defined default (Medium in this case) or Custom.
+
+![template error](img/Step-1-basic-template-2.png)
+
+Every template should be shown within the Template dropdown. If the selected template will not be able to run on the cluster (because the CPU family is incompatible or a Machine Type is unavailable) a warning message should be shown with a link to relevant documentation.
+
+## Provision Sources
+
+### Import
+
+![Import](img/Step-1-basic-import-1.png)
+
+Virtual machines can be imported from a curated list of supported providers.
+
+#### VMWare vCenter
+
+![VMWare](img/Step-1-basic-import-2.png)
+
+The user chooses Import as the Provision Source and then selects VMWare as the Provider.
+
+![vCenter](img/Step-1-basic-import-3.png)
+
+If VMWare is chosen, a new “vCenter instance” field appears. The user clicks it to see a list of previously-connected vCenter instances that have been stored as Kubernetes Secrets. The user selects one.
+
+![vCenter-2](img/Step-1-basic-import-4.png)
+
+When a vCenter instance is selected, the first (alphabetical) virtual machine along with its OS, Flavor, and Workload Profile are automatically selected and filled in but remain disabled.
+
+![vCenter Connection issues](img/Step-1-basic-import-5.png)
+
+If the wizard cannot connect to the vCenter instance using the credentials stored within its Secret, an error message will be displayed. The user will need to fix the Secret’s credentials before continuing.
+
+
+![vCenter new instance](img/Step-1-basic-import-6.png)
+
+The user can also connect to a new vCenter instance by clicking Connect to new instance from within the dropdown.
+
+![Checking credentials](img/Step-1-basic-import-7.png)
+
+New fields for vCenter URL, vCenter Username, vCenter Password, and VM to import appear along with a checkbox to “Remember vCenter credentials” that is unchecked by default.
+
+The vCenter URL field should include a syntax hint with the desired formatting. The user is likely to paste a variety of URLs, so any FQDN the user submits should be validated and automatically corrected before being used to communicate with vCenter’s API.
+
+When the user clicks the “Check” button a text string appears below the button with a spinner to indicate that the credentials are being checked.
+
+If the wizard fails to connect to vCenter using the provided credentials, an error similar to the one above for an invalid vCenter instance should be displayed directly below the vCenter Password field.
```

Comment:
I guess I never made an image for this. Maybe we should just remove this sentence?